.github/workflows/java-maven-openjdk-dependency-submission.yml

```diff
@@ -52,7 +52,7 @@ jobs:
             ${{ runner.os }}-maven-
 
       - name: Submit Dependency Snapshot
-        uses: advanced-security/maven-dependency-submission-action@v3
```

Comment:
Hello @jebeaudet, do you think you could test this update too?

Reply:
Sure, from the release notes it seems the only breaking thing is the name of the report, but we didn't use that afaik

Reply:
https://github.com/coveo/authorizationserver/actions/runs/8020670938/job/21910986360?pr=1910

Reply:
Marche bien leur affaire
```
/usr/bin/docker exec  6c2a38d387a518a6ab933caa14c4caead85dffe3342bb8cdf2639e0a57d25f95 sh -c "cat /etc/*release | grep ^ID"
ReferenceError: require is not defined in ES module scope, you can use import instead
This file is being treated as an ES module because it has a '.js' file extension and '/__w/_actions/advanced-security/maven-dependency-submission-action/v4/dist/package.json' contains "type": "module". To treat it as a CommonJS script, rename it to use the '.cjs' file extension.
    at eval (eval at 5178 (file:///__w/_actions/advanced-security/maven-dependency-submission-action/v4/node_modules/@vercel/ncc/dist/ncc/@@notfound.js:1:1), <anonymous>:1:1)
    at Object.5178 (file:///__w/_actions/advanced-security/maven-dependency-submission-action/v4/node_modules/@vercel/ncc/dist/ncc/@@notfound.js:1:1)
    at __nccwpck_require__ (file:///__w/_actions/advanced-security/maven-dependency-submission-action/v4/webpack/bootstrap:21:1)
    at file:///__w/_actions/advanced-security/maven-dependency-submission-action/v4/dist/index.js:30[9](https://github.com/coveo/authorizationserver/actions/runs/8020670938/job/21910986360?pr=1910#step:5:10)38:35
    at file:///__w/_actions/advanced-security/maven-dependency-submission-action/v4/dist/index.js:30986:3
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at loadESM (node:internal/process/esm_loader:34:7)
    at handleMainPromise (node:internal/modules/run_main:66:[12](https://github.com/coveo/authorizationserver/actions/runs/8020670938/job/21910986360?pr=1910#step:5:13))```

Reply:
https://github.com/advanced-security/maven-dependency-submission-action/issues/65

Reply:
Alright, on ice for now then! Thanks :)